EIPS/eip-7886.md

```diff
@@ -0,0 +1,399 @@
+---
+eip: 7886
+title: Delayed execution
+description: Make blocks statically verifiable by charging coinbase for inclusion costs upfront
+author: Francesco D`Amato (@fradamt), Toni Wahrstätter (@nerolation)
+discussions-to: https://ethereum-magicians.org/t/eip-7886-delayed-execution/22890
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-02-18
+---
+
+## Abstract
+
+This proposal makes (execution) blocks statically verifiable through minimal checks that only require the previous state, but no execution of the block's transactions, allowing validators attest to a block's validity without completing its execution. We allow transactions to be skipped when invalid at execution time, without invalidating the whole block. To ensure that even skipped transactions pay for their resources, the `COINBASE` pays for all inclusion costs upfront (base cost, calldata and blobs), and recovers the costs only when transactions are successfully executed.
```

Comment:
one unmentioned consideration here is that this will require EL clients to bring back or introduce signer management for local payload building